### DISCUSSION TITLE
Cannot figure out migrations

### DISCUSSION THREAD

**Discussion by ryanhowdy** [NativePHP/laravel#394] [Q&A] [answered]
Just started with nativephp. Installed everything just fine and when i run `php artisan native:serve`, it showed the default laravel welcome page just like i thought.

Then i tried to create some db tables and noticed when i rerun native:serve it says 
```
Skipping Database migration while in development.
You may migrate manually by running: php artisan native:migrate
```

So, i try and run `php artisan native:migrate` i get the following error

```
$ php artisan native:migrate

   Illuminate\Database\QueryException 

  could not find driver (Connection: nativephp, SQL: PRAGMA foreign_keys = ON;)
```

I'm confused because the docs say that i don't need to configure anything for the database to work with sqlite, but this sounds like i've missed a step?

**Comment by simonhamp** [comment]
@ryanhowdy can you share this as a repo on GitHub?

**Reply by ryanhowdy** [reply]
Sure... https://github.com/ryanhowdy/desktop-photo-fixer

**Reply by simonhamp** [reply, marked as answer]
Thanks, I've spun up this repo and I'm not seeing the same error... which leads me to think perhaps your local PHP version (which is the one used when running `php artisan native:migrate`) doesn't have SQLite support baked in

What output do you get when you run this?

```shell
php -i | grep "PDO drivers"
```

**Reply by ryanhowdy** [reply]
yep that is it, just shows `PDO drivers => mysql, sqlsrv`
